resources/frameworks.yaml

```diff
@@ -80,15 +80,15 @@ hyperoptsklearn:
 #    verbose: true
 
 mljarsupervised:
-  version: '0.6.0'
+  version: '0.7.12'
```

Comment:
version number should now be set to `stable` in this file.

Reply:
OK, updated.

Reply:
Yes.

Reply:
And I understand that the final version snapshot will be 14th December?